Nodejs/Product/Nodejs/NodejsTools.vsct

```diff
@@ -59,6 +59,10 @@
         <Parent guid="guidNodeToolsWseContextMenuCmdSet" id="idmNpm"/>
       </Group>
 
+      <Group guid="guidMigrateToJspsCmdSet" id="cmdidJspsMigrate">
```

Comment:
Same with VSCT, the command code, etc. It should go in a new Migration vsix or something, completely independent of the rest of the things, if at all possible.

Reply:
Resolved offline -- decided to keep cmd in the same vsix, but will move migrate command to its own .vsct file (see [above comment](https://github.com/microsoft/nodejstools/pull/2452#discussion_r1137853978)).